### DISCUSSION TITLE
Remove code suggestion if user starts typing

### DISCUSSION THREAD

**Discussion by bramgrooten** [community/community#14695] [Copilot Conversations]
Right now, a code suggestion by Copilot only disappears if I hit `Esc`. Although I regularly like the suggestions, I also find myself pressing `Esc` often. Please make the user is also able to choose for another option: letting the code suggestion disappear if the user starts typing him/herself.

**Comment by jansorg** [comment]
Which editor are you using?

**Reply by bramgrooten** [reply]
PyCharm, using Python

**Reply by bramgrooten** [reply]
This seems to work fine now actually! Not sure what was bothering me this morning. I do remember one moment when I tried to `Tab` a function name I copy-pasted to indent it some more, but copilot added some code instead :/ Probably a difficult issue to try to recognize when a user wants to indent or wants to accept the copilot suggestion.
